packages/web/src/dev/ValidatorsListApp.tsx

```diff
@@ -0,0 +1,87 @@
+import ApolloClient from 'apollo-boost'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import fetch from 'cross-fetch'
+import gql from 'graphql-tag'
+import getConfig from 'next/config'
+import * as React from 'react'
+import { ApolloProvider, Query } from 'react-apollo'
+import ShowApolloError from 'src/dev/ShowApolloError'
+import ValidatorsList from 'src/dev/ValidatorsList'
+import { I18nProps, withNamespaces, NameSpaces } from 'src/i18n'
+
+function createApolloClient() {
+  return new ApolloClient({
+    uri: getConfig().publicRuntimeConfig.BLOCKSCOUT.uri,
+    cache: new InMemoryCache(),
+    // TODO: Remove this workaround when the backend service fixes not needed errors
+    fetch: async (...args) => {
+      const response = await fetch(...args)
+      const { data } = await response.json()
+      return new (Response as any)(JSON.stringify({ data }))
+    },
+  })
+}
+
+const query = gql`
+  query ValidatorGroups {
```

Comment:
Not sure if it is working now but we were using this thing (https://github.com/dotansimha/graphql-code-generator) a while ago in the mobile app to get TS types